packages/db/db-client-types.ts

```diff
@@ -16,6 +16,7 @@ export type UpsertUserOutput = {
   id: number
   // whether the upsert inserted a new record (if falsy, it was updated)
   inserted: boolean
+  inserted_at: string
```

Comment:
The inputs use `camelCase`, so perhaps the outputs should too? (i.e. `insertedAt`). IMO style nit and ez to change later to nonblocking

Reply:
Oh.... actually do we really need this `inserted_at` col in the type at all? I don't see it in the return value? https://github.com/web3-storage/web3.storage/pull/1976/files#diff-ab5fce4fc61478316713f5b4fad807e7dac6e2e2100858f37bfb0b084543814fR123

I think I saw in slack that you tried another approach before involving a timestamp, so maybe this is a remanant of that that can be safely removed?

Reply:
oh yeah, that's just cruft from the initial approach - thanks for catching it